util/chunk/column_test.go

```diff
@@ -302,7 +302,8 @@ func (s *testChunkSuite) TestMyDecimal(c *check.C) {
 		}
 		c.Assert(d.Compare(&ds[i]), check.Equals, 0)
 
-		if err := types.DecimalAdd(&ds[i], d, &ds[i]); err != nil {
```

Comment:
See https://github.com/pingcap/tidb/blob/master/types/mydecimal.go#L1500

Reply:
Can we check `from1 != to && from2 != to` in DecimalAdd? It will be more safe.

Reply:
OK, I will add a check there.

Reply:
Update, PTAL